### ISSUE TITLE
ember-google-maps specifies an invalid, malformed or missing addon at relative path 'lib/in-repo-pin-addon'

### ISSUE THREAD

**Issue by synaptiko** [sandydoo/ember-google-maps#94]
### Complete this checklist before opening an issue

*Please read the documentation before opening an issue*. Once you understand how this addon works, you will find that most problems can be solved by consulting the [Google Maps API reference](https://developers.google.com/maps/documentation/javascript/reference/3.exp/).

* [x] I've read the relevant documentation for this addon on [ember-google-maps.sandydoo.me](https://ember-google-maps.sandydoo.me/docs/getting-started).

* [x] I understand how to set options on the components in this addon. Options are not hardcoded. Arguments passed to a component are used to set the options on the map component.

* [x] I've searched the relevant section of the [Google Maps API reference](https://developers.google.com/maps/documentation/javascript/reference/3.exp/).

* [x] I've [searched the closed issues](https://github.com/sandydoo/ember-google-maps/issues?q=is%3Aissue+is%3Aclosed) for similar problems.

--------------------------------------------------------------------------------

### Describe the bug
I have installed the new version of this addon and I started getting the following warning:
```
WARNING: The 'package.json' file for the addon at /home/jprokop/work/bayzat-fe/node_modules/ember-google-maps
  specifies an invalid, malformed or missing addon at relative path 'lib/in-repo-pin-addon'
```

### To Reproduce
Install the addon in the latest version and run the application with `ember serve`

### Expected behaviour
No warning or an explanation how to remove it.

**Comment by sandydoo** [comment]
Hi, @synaptiko, could you tell me what version you're using? Also, more importantly, which versions of ember and ember-cli?

**Comment by sandydoo** [comment]
Ah, nevermind, I know what the problem is. Thanks for reporting, I'll make a new release 👍

**Comment by sandydoo** [comment]
Fixed in [v3.0.4](https://github.com/sandydoo/ember-google-maps/releases/tag/v3.0.4).
